engine/src/test/java/com/datatorrent/stram/plan/logical/LogicalPlanConfigurationTest.java

```diff
@@ -676,18 +659,22 @@ public void populateDAG(DAG dag, Configuration conf)
     dagBuilder.prepareDAG(dag, app, appName);
```

Comment:
Why is Context necessary? Please be consistent and try to shorten the code. Change MockStorageAgent to Mockito.mock(StorageAgent.class).